### ISSUE TITLE
NullPointerException when clicking items inside player's inventory

### ISSUE THREAD

**Issue by Nikox3003** [TriumphTeam/triumph-gui#20]
```
[16:58:58 ERROR]: Could not pass event InventoryClickEvent to mcwarCore v0.0.6-beta
java.lang.NullPointerException: null
        at net.minecodes.mcwar.gui.guis.GuiListener.isGuiItem(GuiListener.java:176) ~[?:?]
        at net.minecodes.mcwar.gui.guis.GuiListener.onGuiClick(GuiListener.java:98) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor47.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Purpur-1118]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Purpur-1118]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Purpur-1118]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2839) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:55) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1336) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1329) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1305) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1149) ~[patched_1.16.5.jar:git-Purpur-1118]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:292) ~[patched_1.16.5.jar:git-Purpur-1118]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by ipsk** [comment]
Which version of the lib are you using?
Also mind showing me the bit of the code where this happens?

**Comment by Nikox3003** [comment]
Lib version: `3.0.2`
Code: https://paste.letcraft.pl/jokuxareli.cs
I do not handle InventoryClickEvent in any other way.

**Comment by ipsk** [comment]
Fixed in b8832bcf379480ba9d52c6ca27986ca7db01467a, it's out for `3.0.3`. Let me know if it works now.

**Comment by Nikox3003** [comment]
Yup, it works now. 
Thank you for quick update!
